Gemfile

```diff
@@ -6,9 +6,10 @@ gem 'middleman-blog'
 gem 'middleman-livereload'
 gem 'middleman-syntax'
 gem 'middleman-deploy', github: 'middleman-contrib/middleman-deploy'
-# gem 'middleman-search_engine_sitemap', '~> 1.3.0'
-# gem 'middleman-robots', '~> 1.1.0'
+gem 'middleman-search_engine_sitemap', github: 'angeloashmore/middleman-search_engine_sitemap'
```

Comment:
@wikimatze The published gem is incompatible with Middleman 4. You can see the changes here: https://github.com/Aupajo/middleman-search_engine_sitemap/pull/7/files

The extension only needed a few small changes (`app.config.index_page` instead of `app.index_page`, for example), but I also needed to update the tests.

Once the updated gem is published, we can get rid of the `github` flag.